packages/blade/src/components/Carousel/_decisions/decisions.md

```diff
@@ -0,0 +1,161 @@
+# Carousel 
+
+Carousel is a UI component that allows the display and navigation of a set of content items, typically images or cards, within a limited space. It is often used to showcase multiple pieces of content in a visually appealing and interactive way.
+
+<img src="./carousel-thumbnail.png" width="380" />
+
+## Design
+
+[Figma Link](https://www.figma.com/file/LSG77hEeVYDk7j7WV7OMJE/Blade-DSL---Components-Guideline?type=design&node-id=2115-975416&mode=design&t=gjEux4FLvrLihxbH-0) to all variants of the Carousel component
+
+## API
+
+### Carousel
+
+| Prop                     | Type                                   | Default                                  | Description                                                                                                                                                                                                  | Required |
+| ------------------------ | -------------------------------------- | ---------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | -------- |
+| children                 | `CarouselItem`                         | `undefined`                              | Carousel slides                                                                                                                                                                                              | ✅        |
+| autoPlay                 | `boolean`                              | `false`                                  | If true, the carousel will autoplay                                                                                                                                                                          |          |
+| visibleItems             | `1,2,3,auto`                           | `1`                                      | Total number of carousel items to show at once, if set to 1,2 or 3 all the CarouselItem's width will remain the same but if set to `auto` the carousel items will take up space responsively                 |          |
```

Comment:
should we make the visibleItems: `1 | 2 | 3 | auto`? (`auto` being the default.)

Not really fond of using `undefined` as a valid value to switch between variants. (auto also makes more sense i think)

Reply:
auto should be default. Majority of the usecases are where 80% area on mobile device is the item and we want to show some part of next and previous item. Also they might also needs some blur effect.